### ISSUE TITLE
calibrate detents

### ISSUE THREAD

**Issue by jojomo234** [flybywiresim/a32nx#5174]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2021-05-05T06:28:31+00:00",
  "ref": "experimental",
  "sha": "ff7f81fbff658ba8a69c77f6a16754ad1bffabea",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

tried to calibrate detents. as soon as I pressed calibrate the throttle in the aircraft started moving up and down on its own. if I try and set toga detent the throttle moved all the way up then back down then back up. if I try to set flex it moves all the way up then back down then back up to full power but not stopping on the flex detent. same goes for CLB, Idle, Reverse. the simulator shows that the throttle quadrant is working as it should and was working before I tried to calibrate 

### Expected behavior

throttle not to swim

### Steps to reproduce

blue calibrate button 

### References (optional)

https://user-images.githubusercontent.com/35584541/122694102-2c91ac80-d1f1-11eb-9f99-84e824426a0c.mp4

### Additional info (optional)

everything working fine until i pressed calibrate 

### Discord Username (optional)

DeathShadows#8577

**Comment by Benjozork** [comment]
experimental is not updated anymore. please use dec

**Comment by jojomo234** [comment]
Reversed back too dev. problem still persist

**Comment by aguther** [comment]
Some questions:
- what hardware do you have?
- how is your hardware bound to the throttle axes (MSFS directly or third party like SPAD.neXt or FSUIPC)?
- what binding in MSFS or third party do you use?
- do you have any other hardware binding to your throttle axes (you video looks like it is)?

The throttle calibration just processes what it get's from MSFS sent as event and this works properly. So it's something on the input side and the reason for my questions.

**Comment by frogmonkey2** [comment]
? does it act erratically?... shifting up and back, vibrating in place the woosh up to FLEX and back, individual maybe? if so did you download the firmware from your Throttle maker, drivers? if your using Thrustmaster for instance mine worked fine till we had multiple updates from ASOBO and FBW and it became erratic. I didn't have the firmware/drivers EVER downloaded till the situation forced me to take a look on their website. when I did all the 'calibrations' reset... etc... all worked fine. you may need to look up some calibration tips on Youtube on these.

**Comment by jojomo234** [comment]
Thank you everyone for your help I would like you update you on the sequence of things I did to resolve it.
So I had previously the basic 3 lever throttle quadrant from saitek. I then purchased the honeycomb bravo throttle and immediately set it up drivers and all for it. It worked just fine until I decided it was time to properly set the detents. like a mentioned above as soon as I hit calibrate it went all nuts on me. Talking with one of the devs on discored told me that that the detents uses the input value projected from the simulator itself. Checked over the sim to make sure no double bindings or any other setting was to be the cause. To no avail the simulator was reading everything correctly. So naturally I started uninstalling things first the addons then FBW itself. Reinstalled FBW as the only mod added, still no solution then it dawned on me that maybe the drivers from saitek could be messing with it. Uninstalled those and boom fixed. I just don't understand how a driver from a quadrant that's not hooked up to my computer at the moment, suddenly bugs out from pressing  a simple button on a msfs mod. Regardless I'm glad it's fixed. I hope this helps anyone in the future
